test/e2e/bots_test.go

```diff
@@ -663,7 +756,14 @@ func runBotTest(t *testing.T,
 		expectedMessage := fmt.Sprintf("%s\n%s", cmdHeader(command), expectedBody)
 
 		botDriver.PostMessageToBot(t, botDriver.SecondChannel().Identifier(), command)
-		err = botDriver.WaitForLastMessageEqual(botDriver.BotUserID(), botDriver.SecondChannel().ID(), expectedMessage)
+		if botDriver.Type() == commplatform.SlackBot {
+			expMsgFn := func(msg string) (bool, int, string) {
+				return strings.Contains(msg, expectedMessage), 0, ""
+			}
+			err = botDriver.WaitForMessagePosted(botDriver.BotUserID(), botDriver.SecondChannel().ID(), 5, expMsgFn)
```

Comment:
it can be just:
```suggestion
			err = botDriver.WaitForMessagePosted(botDriver.BotUserID(), botDriver.SecondChannel().ID(), 5, commplatform.AssertContains(expectedMessage))
```
if you will add in `commplatform/generic.go`:
```go
func AssertContains(expectedMessage string) MessageAssertion {
	return func(msg string) (bool, int, string) {
		return strings.Contains(msg, expectedMessage), 0, ""
	}
}
```

this can be applied in all other places, lines:
- 778
- 845
- 870
- 920
- 959
- 998
- 1029

some of them has also `=` so we can have other func `AssertEqual` if we want to be more precise.